src/Npgsql.DependencyInjection/README.md

```diff
@@ -0,0 +1,43 @@
+Npgsql is the open source .NET data provider for PostgreSQL. It allows you to connect and interact with PostgreSQL server using .NET.
+
+This package helps set up Npgsql in applications using dependency injection, notably ASP.NET applications. It allows easy configuration of your Npgsql connections and registers the appropriate services in your DI container. 
+
+For example, if using the ASP.NET minimal web API, simply use the following to register Npgsql:
+
+```csharp
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddNpgsqlDataSource("Host=pg_server;Username=test;Password=test;Database=test");
+```
+
+This registers a scoped `NpgsqlConnection` which can get injected into your controllers:
+
+```csharp
+public class MyController : Controller
```

Comment:
Who still uses controllers /jk 

We may want to swap one of these samples out for a minimal api one instead.

Reply:
You're right, I'm a bit behind the times when it comes to ASP.NET. Will do.

Reply:
@NinoFloris check out the new samples - looks great. The automatic logging integration is kinda magical!